Abstract/abstract-blackfire-php-extension.rb

```diff
@@ -164,7 +164,7 @@ def config_filepath
   def write_config_file
     if config_filepath.file?
       inreplace config_filepath do |s|
-        s.gsub!(/^(zend_)?extension=.+$/, "extension=\"#{module_path}\"")
+        s.gsub!(/^(;\s*)?(zend_)?extension=.+$/, "extension=\"#{module_path}\"")
```

Comment:
Should it not be

```suggestion
        s.gsub!(/^(;\s*)?(zend_)?extension=.+$/, "\\1extension=\"#{module_path}\"")
```
Otherwise, it explicitly enables the module which has been disabled, so it must be manually disabled again.